Contentless/Program.cs

```diff
@@ -43,11 +44,62 @@ public static class Program {
         var excluded = config.ExcludedFiles.Select(Program.MakeFileRegex).ToArray();
         var overrides = Program.GetOverrides(config.Overrides).ToArray();
 
+        var referencesVersions = config.References.ToList().ToDictionary(x => x, x => (string)null, StringComparer.OrdinalIgnoreCase);
+        if (config.References.Any())
+        {
+            var csprojPath = args[1];
+            Console.WriteLine($"Using project file {csprojPath}");
+            var projectRootElement = ProjectRootElement.Open(csprojPath);
+            foreach (var property in projectRootElement.AllChildren.Where(x => x.ElementName == "PackageReference").Select(x => x as ProjectItemElement))
+            {
+                var libraryName = property.Include;
+                var version = (property.Children.First() as ProjectMetadataElement).Value;
+                if (config.References.Any(x => x.Equals(libraryName, StringComparison.InvariantCultureIgnoreCase)))
+                if (referencesVersions.Keys.Contains(libraryName))
+                {
+                    referencesVersions[libraryName] = version;
+                    //syncingReferences.Add(libraryName, version);
+                    Console.WriteLine($"Found library version for sync: {libraryName}, {version}");
+                }
+            }
+
+            foreach (var library in referencesVersions)
+                if (library.Value is null)
+                {
+                    Console.WriteLine($"Unable to find library {library.Key}");
+                    return;
```

Comment:
Why return here? Can't we just continue anyway like when loading the config fails?

Reply:
It means that a user have setted library in `Contentless.json` and forgot about it in .csproj, so I don't know library's version. Yes, I can ignore this error. 
But, may be better highlights this error without return(for example, error stdout, or red color in console). It also for other errors.

Reply:
I still think returning entirely here is unnecessary. You can use `Console.Error` for your errors if you prefer, I agree that that might make it easier to see. I don't think it should end the program, though!